### PR TITLE
chore: add test coverage commands

### PR DESCRIPTION
**Pull request by fpgmaas**

### CODE REVIEW

**Review by SemyonSinchenko — COMMENTED** [review]
I do not fully understand the idea. Why do we need both html and xml? And why do we generate html by default. Why cannot we just use stdout? With a simple `--cov=chispa` it will print a nice and readable message with summary about coverage.

I would like to see the stdout message on a `make test` and html/xml reports should be generated by `test-coverage` or something like this.

Also, for example, I do not have an `open` command (`xdg-open` should be used instead). What do you think about removing `open` and generate only reports?

**Comment by fpgmaas** [comment]
> I do not fully understand the idea. Why do we need both html and xml? And why do we generate html by default. Why cannot we just use stdout? With a simple `--cov=chispa` it will print a nice and readable message with summary about coverage.
> 
> I would like to see the stdout message on a `make test` and html/xml reports should be generated by `test-coverage` or something like this.
> 
> Also, for example, I do not have an `open` command (`xdg-open` should be used instead). What do you think about removing `open` and generate only reports?

@SemyonSinchenko  Sorry, I could have elaborated a bit more in the PR description. The idea is to have:
- A html report for local development; this allows you to easily see which lines from which file are covered or not covered. A lot more user friendly than xml
- An xml report to add automated test coverage checking on PR's in a later stage, with [codecov](https://about.codecov.io) or a similar tool.

Hope this clarifies! I modified the commands and removed the `open` command for the HTML report.

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,17 @@ install: ## Install the Poetry environment
 .PHONY: test
 test: ## Run unit tests
 	@echo "Running unit tests"
-	@poetry run pytest tests
+	@poetry run pytest tests --cov=chispa --cov-report term
```
Comment (SemyonSinchenko) [inline Makefile:9]:
```suggestion
	@poetry run pytest tests --cov=chispa --cov-report=term 
```

Because in other places we are using it with `=`

**Review by SemyonSinchenko — APPROVED** [review]
LGTM overall. I left one minor comment. Feel free to merge when you are done.

**Pull request merged**